index.html

```diff
@@ -0,0 +1,198 @@
+          </div>
+          <h3 class="program-title">Lecture</h3>
+          <p class="p-1">
+            Listen to the snooker lessons taught by the most-experienced
+            international grandmasters.
+          </p>
+        </article>
+        <article class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-chess-board"></i>
+          </div>
+          <h3 class="program-title">Match</h3>
+          <p class="p-1">
+            Watch the exciting snooker matches by the top players and learn from
+            them.
+          </p>
+        </article>
+        <article class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-chess-queen"></i>
+          </div>
+          <h3 class="program-title">Practice</h3>
+          <p class="p-1">
+            Play snooker with other players over a chessboard, rather than
+            playing online.
+          </p>
+        </article>
+        <article class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-book"></i>
+          </div>
+          <h3 class="program-title">Theory</h3>
+          <p class="p-1">
+            Study the important theory about the openings, middle game and
+            endgame.
+          </p>
+        </article>
+        <article class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-users"></i>
+          </div>
+          <h3 class="program-title">Connect</h3>
+          <p class="p-1">
+            Become part of the amazing snooker community to discuss, study and
+            play.
+          </p>
+        </article>
+      </section>
+      <button type="button" id="join-btn">Join Event</button>
+    </section>
+    <section class="feature-container d-flex flex-column align-items-center p-3 bg-white" id="player-section">
+      <h2 class="section-heading">Featured Players</h2>
+      <hr />
+      <section class="row justify-content-center">
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player1">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player2">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player3">
+   
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player4">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player5">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player6">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player7">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player8">
```

Comment:
- [Optional] It would be nice to implement the `see more` button so that users can see more feature speakers.
![more](https://user-images.githubusercontent.com/69077061/204098862-d7622586-5fcd-467c-b73d-47095d79bec4.PNG)